server/views/pages/recentArrivals/recentArrivals.njk

```diff
@@ -1,16 +1,29 @@
 {% extends "../../partials/layout.njk" %}
-{% from  "../../partials/expectedArrivalCardMacro.njk" import expectedArrivalCard, transferCard  %}
+{% from  "../../partials/expectedArrivalCardMacro.njk" import recentArrivalCard  %}
 {% set pageTitle = "Prisoners who have arrived in the last 3 days" %}
 
 {% block content %}
     <div class="govuk-!-margin-bottom-8">
         <div class="govuk-width-container govuk-body">
             <h1>{{pageTitle}}</h1>
-                <h2 class="govuk-heading-m govuk-!-margin-top-9">Wednesday 27 April</h2>
-
-                <h2 class="govuk-heading-m govuk-!-margin-top-9">Tuesday 26 April</h2>
-
-                <h2 class="govuk-heading-m govuk-!-margin-top-9">Monday 25 April</h2>
+            <h2 class="govuk-heading-m govuk-!-margin-top-9" id='today'>{{dates.today}}</h2>
+                {% for arrival in recentArrivals.get(dates.today) %}
```

Comment:
might be tempted to push this down into the service layer and just iterate over the `Map<Date, List<Move>>`?

The date calculation in the controller could be pushed down too - that will probably help if we need to change this.

Reply:
sorted in latest PR